### DISCUSSION TITLE
S3 PipeDataset Example for Sagemaker

### DISCUSSION THREAD

**Discussion by imshashank** [aws/sagemaker-python-sdk#2454] [Help]
**What did you find confusing? Please describe.**
I have been trying to start a Sagemaker job with s3pipedataset using TensorFlow or PyTorch but so far I have not been successful. Also, I think pipe mode doesn't work locally.

Is there some documentation or a notebook for end-to-end training using s3pipedataset?

Or at least provide some datasets already stored in PipeDatasetFormat that can be directly loaded into a TensorFlow or PyTorch model.

If possible can you guys also share some Image Dataset for the Computer Vision example?

A clear and concise description of what you found confusing. Ex. I tried to [...] but I didn't understand how to [...]

**Describe how documentation can be improved**
A clear and concise description of where documentation was lacking and how it can be improved.

**Additional context**
Add any other context or screenshots about the documentation request here.

**Comment by cfregly** [comment]
https://github.com/data-science-on-aws/workshop/blob/749a1b/07_train/src/tf_bert_reviews.py#L62
